compiler-core/src/language_server/engine.rs

```diff
@@ -613,3 +640,57 @@ fn code_action_unused_imports(
         .preferred(true)
         .push_to(actions);
 }
+
+fn get_expr_qualified_name(expression: &TypedExpr) -> Option<(&EcoString, &EcoString)> {
```

Comment:
What is `module_name` here? Is it the name or the alias of the imported module? I'm not totally sure but I think this function mixes them up.

Reply:
it should be the original name... there are tests to confirm that